modules/apps/foundation/configuration-admin/configuration-admin-web/src/main/resources/META-INF/resources/configuration_category_menu.jsp

```diff
@@ -61,6 +61,7 @@ ResourceBundleLoaderProvider resourceBundleLoaderProvider = (ResourceBundleLoade
 							<li class="nav-item">
 								<portlet:renderURL var="editURL">
 									<portlet:param name="mvcRenderCommandName" value="/edit_configuration" />
+									<portlet:param name="redirect" value="<%= currentURL %>" />
```

Comment:
This is not a good idea because then, the back button at the top goes back to the previously selected entry in the menu, instead of going back to the front page.

Why did you feel the need to add it?

Reply:
I did this because of the search results. After a user performs a search and select an entry, the back button goes back to the search results page in the old design. I think that behavior is more user friendly because if a user is searching for a configuration, the user might not get what s/he needs on the first click, and might need to go through a couple entries in the search results to find it.

I agree it's not a good idea if this also takes you back to the previously selected entry in the menu. In this case, it's a UX design call. I'm okay with changing it back to always go back to the front page.

Reply:
That's a good point. I'll try to think of a solution that solves that without the problem I mentioned.